[2024-10-29T22:19:41.575+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: count_a_dag.generate_files manual__2024-10-29T22:19:37.484795+00:00 [queued]>
[2024-10-29T22:19:41.700+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: count_a_dag.generate_files manual__2024-10-29T22:19:37.484795+00:00 [queued]>
[2024-10-29T22:19:41.701+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-10-29T22:19:41.752+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): generate_files> on 2024-10-29 22:19:37.484795+00:00
[2024-10-29T22:19:41.759+0000] {standard_task_runner.py:57} INFO - Started process 225 to run task
[2024-10-29T22:19:41.763+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'count_a_dag', 'generate_files', 'manual__2024-10-29T22:19:37.484795+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/count_a_dag.py', '--cfg-path', '/tmp/tmp3dg9shb0']
[2024-10-29T22:19:41.767+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask generate_files
[2024-10-29T22:19:41.938+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-29T22:19:42.115+0000] {task_command.py:410} INFO - Running <TaskInstance: count_a_dag.generate_files manual__2024-10-29T22:19:37.484795+00:00 [running]> on host cf88f9d874cd
[2024-10-29T22:19:42.873+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='user' AIRFLOW_CTX_DAG_ID='count_a_dag' AIRFLOW_CTX_TASK_ID='generate_files' AIRFLOW_CTX_EXECUTION_DATE='2024-10-29T22:19:37.484795+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-29T22:19:37.484795+00:00'
[2024-10-29T22:19:43.164+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-29T22:19:43.234+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=count_a_dag, task_id=generate_files, execution_date=20241029T221937, start_date=20241029T221941, end_date=20241029T221943
[2024-10-29T22:19:43.302+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-29T22:19:43.487+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
